_posts/2019-09-05-community-update.md

```diff
@@ -0,0 +1,146 @@
+
+Flink continues to push the boundaries of <strike>stream</strike> data processing, and the community is proud to see an ever-increasingly diverse set of contributors, users and technologies join the ecosystem. 
+
+In the timeframe of three releases, the project jumped from **112 to 190 contributors**, also doubling down on the number of requested changes and improvements. To top it off, the Flink GitHub repository recently reached the milestone of **10k stars**, all the way up from the incubation days in 2014.
+
+<center>
+<img src="{{ site.baseurl }}/img/blog/2019-09-05-flink-community-update/2019-09-05-flink-community-update_1.png" width="1000px" alt="GitHub"/>
+</center>
+
+The activity across the @user and @dev<sup>1</sup> Mailing Lists shows a healthy heartbeat, and the gradual ramp up of @user-zh suggests that this was a well-received community effort. Looking at the numbers for the same period in 2018, the @dev Mailing List has seen the biggest surge in activity, with an average growth of **2.5x in the number of messages and distinct users** — a great reflection of the hyperactive pace of development of the Flink codebase.
+
+<img style="float: right;" src="{{ site.baseurl }}/img/blog/2019-09-05-flink-community-update/2019-09-05-flink-community-update_2.png" width="420px" alt="Mailing Lists"/>
+
+In support of these observations, the report for the financial year of 2019 from the Apache Software Foundation (ASF) features Flink as one of the most thriving open source projects, with mentions for: 
+
+* Most Active Visits and Downloads
+<p></p>
+* Most Active Sources: Visits
+<p></p>
+* Most Active Sources: Clones
+<p></p>
+* Top Repositories by Number of Commits
+<p></p>
+* Top Most Active Apache Mailing Lists (@user and @dev)
+
+Hats off to our fellows at Apache Beam for an astounding year, too! For more detailed insights, check the [full report](https://s3.amazonaws.com/files-dist/AnnualReports/FY2018%20Annual%20Report.pdf).
+<p></p>
+<sup>1. Excluding messages from "jira@apache.org".</sup>
+
+# Upcoming Flink Community Events
+
+As the conference and meetup season ramps up again, here are some events to keep an eye out for talks about Flink and opportunities to mingle with the wider stream processing community.
+
+### North America
+
+* [Conference] **ApacheCon North America 2019 (September 9-12)** - Las Vegas, USA
+
+* [Conference] **Beam Summit North America 2019 (September 11-12)** - Las Vegas, USA
+
+* [Conference] **Strata Data Conference 2019 (September 23-26)** - New York, USA
+  <p></p>
+  <p></p>
+* [Meetup] **Apache Flink Bay Area Meetup (September 24)** - San Francisco, US ([RSVP](https://www.meetup.com/Bay-Area-Apache-Flink-Meetup/events/262680261/))
+	<p></p>
+	* "Two years of Flink @ Yelp" (Enrico Canzonieri)
+	<p></p>
+	* "How BNP Paribas Fortis uses Flink for real-time fraud detection" (David Massart)
+  <p></p>
+* [Conference] **Scale By The Bay 2019 (November 13-15)** - San Francisco, USA
+
+### Europe
```

Comment:
Maybe mention the CFP for Big Data Tech Warsaw?

Reply:
Good idea!